devito/finite_differences/derivative.py

```diff
@@ -73,6 +73,14 @@ class Derivative(sympy.Derivative, Differentiable):
 
     >>> u.dx2
     Derivative(u(x, y), (x, 2))
+
+    Derivative object are also callable to change default setup:
```

Comment:
probably Function and TimeFunction docstrings to be updated too?

Reply:
Why? This is  a Derivative feature.

Reply:
because I was still sleepy when I wrote it this morning. Ignore me